docs/compute-resources.md

```diff
@@ -45,7 +45,7 @@ Therefore, the pod will have no effective CPU limit.
 ## Task-level Compute Resources Configuration
 
 **([alpha only](https://github.com/tektoncd/pipeline/blob/main/docs/install.md#alpha-features))**
-(This feature is under development and not functional yet. Stay tuned!)
+(This alpha feature is implemented)
```

Comment:
sorry I should have been more clear-- this line can just be removed. (Users can safely assume everything in our docs is implemented unless specified otherwise.